### ISSUE TITLE
Warning alert not emitted when warning audio cue not emitted

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#203211]
Testing #203089

"Audio Cues: Line Has Warning" is set to false. "Accessibility > Alert: Warning" is set to true.

I generated a warning by using the Markdown Lint extension and writing text into a new Markdown file without an h1 header. When I turn on the audio cue, the accessibility alert also alerts me that there's a warning. When I turn off the audio cue but leave the accessibility alert on, the accessibility alert does not alert me that there's a warning.

**Comment by meganrogge** [comment]
I cannot seem to trigger a warning - can you pls give more specific instructions about how to do so?

![Image](https://github.com/microsoft/vscode/assets/29464607/983ed799-c339-42f3-b033-b5282de824b5)

**Comment by rzhao271** [comment]
Install the [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) extension.
Then create a .md file and paste in the following
```

Hello world!

Another line here
```
The extension should already generate a warning.

**Comment by meganrogge** [comment]
Nice find, this is because the audio cue line feature contribution currently only checks the audio cue enabled cache - not the alert enabled cache. 

https://github.com/microsoft/vscode/blob/856d46ed86cacc939504e0d9d77fb8dfc12f9efc/src/vs/workbench/contrib/audioCues/browser/audioCueLineFeatureContribution.ts#L50

Thus, ATM, in order to hear the alerts for line related cues, users have to have the cues enabled. 

@hediet, I plan to refactor the audio cues so that the setting encapsulates both the cues and alerts in February. Thus, I don't think it's worth fiixng this for December/ January atm and it would just complicate things to do so. Do you agree?

**Comment by meganrogge** [comment]
Note this issue when verifying #203266. When I tested using a screen reader, the warning only appeared once. Based on debugging (the `alert` call happens), I believe when using a braille device, this works.
